ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2020.3.8f1
-m_EditorVersionWithRevision: 2020.3.8f1 (507919d4fff5)
+m_EditorVersion: 2020.3.12f1
```

Comment:
Is this wanted or here by mistake?

Reply:
Bumping this. Was a version update necessary?

Reply:
For UTP + Relay integration, I bumped this up to .12f1 but I don't think is necessary